share/spice/sentiment_analysis/document/sentiment_analysis_document.js

```diff
@@ -0,0 +1,77 @@
+var ddg_spice_sentiment_analysis = {
```

Comment:
This is the old approach used for callbacks -- we can simplify things.

Please copy the approach used here: https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/share/spice/amazon/amazon.js#L48

This major difference is that you use the `onShow()` event to trigger a `$.getJSON()` call to your second endpoint. The callback function provided should take the relevant info and use `item.set('property', value)` to update the item as needed. This will force the UI to update as the data for the item is modified.

This way we also avoid defining multiple callback functions/context switches etc.

Don't forget to wrap everything in a closure and add your callback as a property of `env` -- this ensures you can define private functions and variables without polluting the global namespace.